AdyenActions/Components/QRCode/QRCodeComponent.swift

```diff
@@ -197,8 +216,8 @@ extension QRCodeComponent: ActionComponentDelegate {
 /// :nodoc:
 extension QRCodeComponent: QRCodeViewDelegate {
     
-    func copyToPasteboard() {
-        UIPasteboard.general.string = action.map(\.qrCodeData)
+    func copyToPasteboard(with action: QRCodeAction) {
```

Comment:
this should probably have access level specifier

Reply:
can I fix this in a new PR?

Reply:
yes, i approved this one :)